src/specific/music.c

```diff
@@ -43,14 +43,14 @@ int32_t MusicInit()
     return 1;
 }
 
-void S_MusicVolume(int16_t volume)
+void S_Music_AdjustVolume(int16_t volume)
```

Comment:
Adjust implies a relative operation, this is an absolute function. S_Music_SetVolume

Reply:
Point, fixed